frontend/packages/ceph-storage-plugin/integration-tests/tests/installFlow.scenario.ts

```diff
@@ -0,0 +1,254 @@
+import { execSync } from 'child_process';
+import { ExpectedConditions as until, browser } from 'protractor';
+import * as crudView from '@console/internal-integration-tests/views/crud.view';
+import {
+  goToOperatorHub,
+  installOperator,
+  InstallCluster,
+  searchInOperatorHub,
+  uninstallOperator,
+  closeOperatorInstall,
+  storageClusterLink,
+  btnCreate,
+  selectWorkerRows,
+  status,
+  NS,
+  operatorsPage,
+  ocsOperator,
+  ocsOperatorStatus,
+  filterInput,
+  getDataFromRowAndCol,
+  podNameFilter,
+  goToStorageClasses,
+  checkIfClusterIsReady,
+  getPodData,
+  getPodPhase,
+  getPodName,
+} from '../views/subscriptionFlow.view';
+
+const installer = new InstallCluster(NS);
+
+const ocsDeploy = `
+---
+apiVersion: v1
```

Comment:
namespace creation flow should indeed be tested. But if those tests are already intact and it proves namespace creation component works. Are you worried that namespace creation flow might be something that can fail? What's the reasoning behind it. If it indeed can fail then it is a bug.

Reply:
Are you telling that everyone will always run all the tests in the test suites?
I don't think so. So, if I choose to run only `ceph-storage` suite, who is responsible for testing the very first step of our documentation?

Or, are you telling that since OCP does the tests on their side, it will always work?
If this is the case, I'd say that's too much of optimism :) Never assume that it'll work.

Reply:
yes, namespace creation is the very first test. So for some reason, if namespace creation flow has an issue. It will always fail first. In fact, this test would not be possible if the namespace creation flow fails. I am not assuming it's just a logical conclusion that if a test namespace can be created and that test passes why is there a need to verify the same component again in the same run?

Reply:
It's axiomatic that if for some reason creation of test namespace UI flow passes and for some reason creation of `openshift-storage` namespace fails. Then the UI flow is perfectly fine and there are some issues in the backend or the cluster setup. Which will be caught even if we perform it using kubectl since it will also fail.